dprof.js

```diff
@@ -60,7 +60,9 @@ function Node(uid, handle, name, stack, parent) {
   this._destroy = Infinity;
   this._before = [];
   this._after = [];
-  this.unrefed = this.name === 'TTY' || this.name === 'Pipe' || handle._timerUnref === true;
+  this._unref = [];
+  this._ref = [];
+  this.unrefed = this.name === 'TTYWRAP' || this.name === 'PIPEWRAP' || handle._timerUnref === true;
```

Comment:
this PR relies on AsyncHooks and targets that branch... I'll move it into that PR later